app.py

```diff
@@ -0,0 +1,283 @@
+import os
+import os.path as op
+from flask import Flask
+from flask import request
+from flask_sqlalchemy import SQLAlchemy
+import africastalking
+
+import flask_admin as admin
+from flask_admin.contrib.sqla import ModelView
+
+app = Flask(__name__)
+app.config.from_pyfile('config.py')
+db = SQLAlchemy(app)
+
+username = "sandbox"   
+api_key = os.environ.get('AT_API_KEY')   
+test_number = "+254723963007"  
```

Comment:
is this a test number they provided?

Reply:
nope, their setup allows you to input any number for testing with the sandbox environment. I think I just grabbed that off wikipedia as an example number. I can make it more generic, I think it just has to have the 254 at the start

Reply:
Awesome :) while I'm in Kenya we can change that to my phone number for testing :)